### ISSUE TITLE
How to import this module into my vue project？

### ISSUE THREAD

**Issue by zhengzhi530** [dekyfin/RichFilemanager-NODE#16]
hi master,
   I was used it on a seperate server from your front-end but i don't really know how to import these js file into my .vue files? can you give me a case if it's convenient for you.
   I expect for your reply.
   Thanks！

**Comment by jlaustill** [comment]
I don't know if anyone has used this with vue yet, but that would be a great example to have.

**Comment by zhengzhi530** [comment]
Okay, I'll try it and thanks again.

**Comment by zhengzhi530** [comment]
@jlaustill  Yes,thanks man.vue is the mainstream development framework in our region.
